tests/bench_xtimer/Makefile.ci

```diff
@@ -0,0 +1,3 @@
+BOARD_INSUFFICIENT_MEMORY := \
```

Comment:
You don't need to add this file for this test. Just add the board in the `LOW_MEMORY_BOARDS` variable.

https://github.com/RIOT-OS/RIOT/blob/0703449c0f58c082ed9dbc0253b89cc6e7ba2992/tests/bench_xtimer/Makefile#L7